server/aws-lsp-codewhisperer/src/client/streamingClient/codewhispererStreamingClient.ts

```diff
@@ -4,7 +4,7 @@ import { readFileSync } from 'fs'
 
 const codeWhispererRegion = 'us-east-1'
 
-const codeWhispererEndpoint = 'https://codewhisperer.us-east-1.amazonaws.com/'
+const codeWhispererEndpoint = process?.env.AWS_Q_ENDPOINT_URL ?? 'https://codewhisperer.us-east-1.amazonaws.com/'
```

Comment:
we can configure this URL in some util file, so we don't need to change in multiple files.

Reply:
done